requirements.txt

```diff
@@ -1,27 +1 @@
-f90nml
-pytest-xdist
-pytest-dependency
-pytest
-pytest-cov
-pytest-subtests
-pytest-profiling
-xarray
-numpy
-zarr
-cftime
-git+https://github.com/mpi4py/mpi4py.git@aac3d8f2a56f3d74a75ad32ac0554d63e7ef90ab
-pip-tools
-wheel
-netCDF4
-hypothesis
-scipy
-git+https://github.com/VulcanClimateModeling/fv3config.git@a0799928824a4f10617cf4615e8837c81ccb1f2d
-pytest-regressions
-gitpython
-
-# GT4Py Tests
-clang-format
-factory-boy
-pytest-factoryboy
-pytest-cache
-pytest-cov
+-r requirements/requirements_container.txt
```

Comment:
it's a little confusing for the central requirements.txt to be specifically for the container, do we need this file at all?

Reply:
Well, it is more that the central file should be the one that also works in most cases. And the thing we do in containers is what works locally, it is just on daint - with modules etc, we need something differently. I am open to renaming it if you think the intent gets clearer?

Reply:
and going this route the "default" behaivour we had so far does not change

Reply:
ok we can leave it then!